components/ThemeToggler.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="theme-toggle-wrapper" @click="toggleTheme()">
+    <span>
+      <img class="theme-toggle-icon" :src="require(`~/assets/${themeImage}`)" />
+    </span>
+  </div>
+</template>
+
+<script>
+function setCookie(name, value, days) {
+  let expires = ''
+  if (days) {
+    const date = new Date()
+    date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000)
```

Comment:
Just for the information: How are we getting days here 
Is it not usually 28-31 days

Reply:
Yes it is, but as per the discussion we decided to go for 30 days expiry